service/static/js/rest_api.js

```diff
@@ -246,4 +262,84 @@ $(function () {
         });
 
     });
+
+    $("#clear-item-btn").click(function () {
+        $("#item_id").val("");
+        $("#item_shopcart_id").val("");
+        $("#item_name").val("");
+        $("#item_quantity").val("");
+        $("#item_price").val("");
+        $("#flash_message").empty();
+        clear_form_shopcart()
+    });
+
+    $("#search-item-btn").click(function () {
```

Comment:
I think we have only search shopcarts API (e.g. search shopcarts by name in the List Shopcarts API)?

Reply:
The button for item search can be eliminated I think 🙂

Reply:
But we have this API either, "GET  /shopcarts/{shopcart_id}/items"

Reply:
@LillisLuo sorry for my belated reply and you're right, the button should not be deleted because we still have the List Shopcart Items API lol

Reply:
Hence I will mark this part as resolved. Thank you again for the reminder!